packages/server/src/tests/banners/ChannelBannerTests.ts

```diff
@@ -37,6 +38,8 @@ export const BannerPaths: {
     [BannerTemplate.InvestigationsMomentBanner]: investigationsMomentBanner.endpointPathBuilder,
     [BannerTemplate.EnvironmentMomentBanner]: environmentMomentBanner.endpointPathBuilder,
     [BannerTemplate.UsEoyMomentBanner]: usEoyMomentBanner.endpointPathBuilder,
+    [BannerTemplate.UsEoyMomentGivingTuesdayBanner]:
+        usEoyMomentGivingTuesdayBanner.endpointPathBuilder,
```

Comment:
We'll need to update our hardcoded logic to get the ticker to show up (i.e on line 78)

Reply:
Yes! Have now extended that logic on line 78, was a bit thrown off by the banner showing in the live preview but not the web preview in the tool